pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -212,6 +212,8 @@ class Build {
             } else if (buildConfig.VARIANT == "openj9") {
                 jdkBranch = 'openj9'
             } else if (buildConfig.VARIANT == "hotspot"){
+                jdkBranch = 'master'
```

Comment:
Does this mean hotspot variant is 'openjdk'?

Reply:
I think this is ok as both temurin and hotspot would go into the `variant = "hs"` section which is correct. I do NOT plan to build `VARIANT=hotspot` in jenkins, so at the moment the `hs` test jobs would be exclusively for the `temurin` variant and when I was testing last week with this change the `hs` jobs got invoked correctly. We may wish to change that later of course but there should be no reason to change from `hs` just now :-)